.github/workflows/build.yml

```
@@ -368,6 +368,14 @@ jobs:
         echo "HYPOVENV=${VENV_LOC}" >> $GITHUB_ENV
         echo "VENV_PYTHON=${VENV_PYTHON}" >> $GITHUB_ENV
         ./python -m venv $VENV_LOC && $VENV_PYTHON -m pip install -U hypothesis
+    - name: 'Restore Hypothesis database'
+      id: cache-hypothesis-database
+      uses: actions/cache@v3
+      with:
+        path: ./hypothesis
+        key: hypothesis-database-${{ github.head_ref || github.run_id }}
+        restore-keys:
+          - hypothesis-database-
```

Something like this?
```suggestion
        restore-keys: |
          hypothesis-database-
```
https://github.com/actions/cache/blob/main/examples.md#python---pip

---

Ugh, yep, that's the trick.  

https://github.com/actions/cache/#inputs is just entirely wrong in calling for `An ordered list of prefix-matched keys`; it's actually a *multi-line string of keys*.  Which is a design choice, I guess!